cider/include/cider/CiderCompileModule.h

```diff
@@ -93,6 +94,7 @@ class CiderCompileModule {
 
   class Impl;
   std::unique_ptr<Impl> impl_;
+  std::shared_ptr<CiderAllocator> allocator_;
```

Comment:
where is it from? via constructor?

Reply:
> where is it from? via constructor?

Yes, and i have changed the related code just now